inc/saiport.h

```diff
@@ -1293,7 +1312,14 @@ typedef enum _sai_port_attr_t
      * @default internal
      */
     SAI_PORT_ATTR_PORT_SERDES_ID,
-
+    
+    /** @brief Link training failure status and error codes
```

Comment:
move @brief to next line, and add empty line before type, please look at style on other attributes

Reply:
done